### ISSUE TITLE
allow some classes to have multiple turns in combat

### ISSUE THREAD

**Issue by seiyria** [IdleLands/IdleLands#613]

**Comment by Kookiecraft** [comment]
Perhaps give every class a dex threshold where they begin to attack multiple times per turn?

**Comment by seiyria** [comment]
I like that. Then dex becomes more valuable, and dex classes inherently do this. maybe get +1 attack every 10k dex?

**Comment by Kookiecraft** [comment]
I was thinking of a different value for each class. Dex helps with some of an archer's damage, so the threshold for archer should probably be higher than the other classes for instance.

**Comment by seiyria** [comment]
True. I think that's reasonable as well. Additionally fighters should
probably never get it due to their skillset. Or maybe the skillset is
phased out in favor of this.
